examples/island/Island.cpp

```diff
@@ -11,6 +12,7 @@
 #include <algorithm>
 #include <cstring>
 #include <cmath>
+#include <iostream>
```

Comment:
I would move this include above `<cstring>`.

Reply:
Will fix